app/src/main/java/com/bootcamp_android/parking_app/adapter/LotViewHolder.kt

```diff
@@ -16,12 +16,19 @@ class LotViewHolder(view: View):RecyclerView.ViewHolder(view) {
     fun binding(lotModel : Lot){
 
         id.text = lotModel.id
-        date.text = lotModel.date
+        if(lotModel.available) {
 
-        hour.text = lotModel.hour
+            date.text = lotModel.date
 
```

Comment:
Do not let blank spaces between the code inside of the method
Try to use the "Formate Code" of the IDE